google/cloud/aiplatform/metadata/artifact.py

```diff
@@ -32,7 +32,7 @@
 from google.cloud.aiplatform.metadata import resource
 from google.cloud.aiplatform.metadata import utils as metadata_utils
 from google.cloud.aiplatform.utils import rest_utils
-
+from google.cloud.aiplatform.metadata.schema import base_artifact
```

Comment:
This should be in  lexicographical order in the import block

Reply:
Done.